index.html

```diff
@@ -312,8 +312,11 @@ <h3>
             time immediately before the user agent <a data-cite=
             "service-workers#fetchevent">fired an event named `fetch`</a> at
             the <a data-cite="service-workers#dfn-active-worker">active
-            worker</a>. Otherwise, if there is no active worker this attribute
-            MUST return zero.
+            worker</a>. If there were redirects, this is the start time of
```

Comment:
Do you know if this is the behavior of user agents right now? In particular, does this require new tests?

Reply:
Reviewing current WPT tests, I think we'll need to add some to confirm this all